src/game/hooks/GUI/Window.cpp

```diff
@@ -10,9 +10,6 @@ namespace YimMenu::Hooks
 		if (g_Running)
 			Renderer::WndProc(hwnd, umsg, wparam, lparam);
 
-		if (umsg == WM_KEYUP && wparam == VK_DELETE) // fallback to unload
```

Comment:
This button was useful when a certain UI change during development resulted in the UI button being unusable.

Reply:
I'm guessing unloading online raises detections concerns.